test/support/test_repo/migrations/20230209214749_create_test_schema_with_prefix.exs

```diff
@@ -0,0 +1,24 @@
+defmodule BitcrowdEcto.TestRepo.Migrations.CreateTestSchemaWithPrefix do
+  use Ecto.Migration
+
+  def change do
+    prefix = "foo"
```

Comment:
Kind of liked your `alternate_test_schema` in the other project :upside_down_face:

Reply:
want a longer prefix name?
I actually made a migration for something like `test_schema_using_prefix` but then realized i could use the same schema if i named it the same..